sea-asean-covid-19-mortality-rate/index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>SEA COVID-19 Mortality Rate </title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- Include Project Fonts -->
+  
+</head>
+<body>
+  <!-- Highcharts chart container -->
+  <div id="hcContainer"></div>
+
+  <!-- Include jQuery & Highcharts Libraries; Adjust per project -->
+  <script src="https://code.highcharts.com/highcharts.js"></script>
+  <script src="https://code.highcharts.com/highcharts-more.js"></script>
+  <script src="https://code.highcharts.com/modules/data.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+  <!-- Include iFrame Resizer -->
+  <script src="//csis-ilab.github.io/cpower-viz/globaljs/iframeSizer.contentWindow.min.js"></script>

```

Comment:
```suggestion
  <script src="https://csis-ilab.github.io/cpower-viz/globaljs/iframeSizer.contentWindow.min.js"></script>
```

Remember, these need to have their full protocol included.